src/scripts/content/workflowy.js

```diff
@@ -1,82 +1,35 @@
 'use strict';
 /* global togglbutton, $ */
 
-/* Task more menu (formerly bullet hover) */
 togglbutton.render(
-  '.name:not(.toggl) div:not(.toggl) > .flyout.menu',
+  '.header:not(.toggl)',
   { observe: true },
   $container => {
-    const $bulletInfo = $('.content', $container.closest('.name'));
-
-    const descriptionSelector = () => {
-      return getDescription($bulletInfo);
-    };
-
-    const tagsSelector = () => {
-      return getTags($bulletInfo);
-    };
-
     const link = togglbutton.createTimerLink({
       className: 'workflowy',
-      description: descriptionSelector,
-      tags: tagsSelector
+      buttonType: 'minimal',
+      description: getDescription,
+      tags: getTags
     });
 
-    insertLink($container, link);
-  }
-);
-
-/* More Options Menu Popup */
-togglbutton.render(
-  '.pageMenu:not(.toggl) > .flyout.menu',
-  { observe: true },
-  $container => {
-    // Get the content of the top bullet.
-    const $bulletInfo = $('.content');
+    $container.insertBefore(link, $container.children[3]);
 
-    const descriptionSelector = () => {
-      return getDescription($bulletInfo);
-    };
-
-    const tagsSelector = () => {
-      return getTags($bulletInfo);
-    };
-
-    const link = togglbutton.createTimerLink({
-      className: 'workflowy',
-      description: descriptionSelector,
-      tags: tagsSelector
+    document.addEventListener('focusin', function (e) {
```

Comment:
I think we should try to be good citizens here, and ensure there's only one listener added. It's possible the `togglbutton.render` callback will run multiple times over the lifetime of the browser tab and add many listeners.

One option might be to simply move `document.addEventListener..` to the root of the file?

Reply:
Done!